docker/kubernetes/README.md

```diff
@@ -0,0 +1,256 @@
+# Running the Gameroom Demo in Kubernetes
+
+This procedure explains how to run the
+[Gameroom demo](https://github.com/Cargill/splinter/tree/master/examples/gameroom)
+with
+[Kubernetes](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/).
+This environment uses [Minikube](https://kubernetes.io/docs/setup/minikube/) to
+deploy two containerized Splinter nodes in a local Kubernetes cluster inside a
+virtual machine (VM) on your computer.
+
+This procedure walks you through:
+
+* Generating keys and updating the key registry
+* Creating ConfigMaps for the key registry and node registry
+* Starting Gameroom
+* Creating users and logging in to the web app
+* Playing tic-tac-toe
+
```

Comment:
For consideration, I'd like to see a caveat or call out in the README that states "this single pod deployment for each organization isn't intended to depict a production-like scenario. This is to demo the Kubernetes primitives." ... or something.